event/generate.go

```diff
@@ -138,15 +145,20 @@ func main() {
 		e.Name = link.Text()
 		e.Link, _ = link.Attr("href")
 		e.Desc = strings.TrimSpace(cols.Eq(3).Text())
-		if e.Desc == "" {
-			e.Desc = "(no documentation)"
-		}
+		e.Spec = strings.TrimSpace(cols.Eq(2).Text())
 
 		funName := nameMap[e.Name]
 		if funName == "" {
 			funName = capitalize(e.Name)
 		}
 
+		if e.Desc != "" {
+			// Lowercase the first letter of the description.
+			lowercaseDesc := strings.ToLower(string(e.Desc[0])) + e.Desc[1:]
```

Comment:
It seems inconsistent that `capitalize` is also used in just one place, yet it's made into a helper, but the same isn't done for its lower equivalent.